plugins/jenkins/src/components/Cards/Cards.tsx

```diff
@@ -40,13 +40,13 @@ const WidgetContent = ({
   const classes = useStyles();
   if (loading || !latestRun) return <LinearProgress />;
   const displayDate = DateTime.fromMillis(latestRun.timestamp).toRelative();
-  // TODO This works, but hard codes as minutes. Would prefer something smarter/relative.
-  const durationInMinutes = Math.round(
-    Duration.fromMillis(latestRun.duration).as('minutes'),
-  );
-  const displayDuration = `${durationInMinutes} minute${
-    durationInMinutes > 1 ? 's' : ''
-  }`;
+  const displayDuration =
+    (latestRun.building ? 'Running for ' : '') +
+    DateTime.local()
+      .minus(Duration.fromMillis(latestRun.duration))
+      .toRelative()
```

Comment:
```suggestion
      .toRelative({ locale: "en" })
```
I think we should override the locale here.
Otherwise we'll have some weird cases where the string will be formatted in the user's language.